### PR TITLE
Added LWIMIQ entry

### PR DESCRIPTION
**Pull request by romanblanco**

### CODE REVIEW

**Comment by JPrause** [comment]
Thanks @romanblanco 
@syncrou @imtayadeway @jntullo can you review if you can spare the time.

File: site/_posts/2017-11-06-call-me-gapri.md [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+[Roman](https://github.com/romanblanco) from UI team here, with my second LWIMIQ article. Lets see what happened last week.
+
+## Improved
+
+There is no week where we wouldn't improve at least a small thing :)
+Last week [Bill](https://github.com/bzwei) has improved [Ansible playbook logging](https://github.com/ManageIQ/manageiq/pull/16333), [Piotr](https://github.com/pkliczewski) has [made the Host trigger refresh after being removed](https://github.com/ManageIQ/manageiq-providers-ovirt/pull/127) and [Shmuel](https://github.com/smelamud) has [extended dialog to select VM by tag](https://github.com/ManageIQ/manageiq-providers-ovirt/pull/115)
+
+## Fixed
+
+With new features may also unfortunately come bugs. Last week [Adam](https://github.com/agrare) has helped ManageIQ to be less buggy by [fixing SCVMM credential validation](https://github.com/ManageIQ/manageiq-ui-classic/pull/2575), [David](https://github.com/ManageIQ/manageiq-ui-classic/pull/2573) updated GTL lists to [use custom pictures](https://github.com/ManageIQ/manageiq-ui-classic/pull/2573) and [Drew](https://github.com/d-m-u) fixed [Custom buttons with subclasses](https://github.com/ManageIQ/manageiq/pull/16378)
```
Comment (JPrause) [inline site/_posts/2017-11-06-call-me-gapri.md]: Did you mean to instead use David's gitter profile?

**Comment by JPrause** [comment]
Funny title,...I could see how it might be stressful to some. 👍

File: site/_posts/2017-11-06-call-me-gapri.md [hunk 2]
```diff
@@ -0,0 +1,52 @@
+---
+title: "Last Week in ManageIQ: Please call me Gapri if it helps you relax"
+author: Roman Blanco
+date: 2017-11-07
+comments: true
+published: true
+tags: LWIMIQ
+---
+
+Wohoo! It's here again! All the features we were working on in last truly _Fine_ summer, are now released in new version, **Gaprindashvili**.
```
Comment (jntullo) [inline site/_posts/2017-11-06-call-me-gapri.md]: I like this 😄  perhaps it should be slightly reworded to "working on over the truly fine summer" as opposed to "in last"?

File: site/_posts/2017-11-06-call-me-gapri.md [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+
+## Deleted
+
+As much important as adding and improving new features is also removing the unused and duplicated code.
+Last week, [Brandon](https://github.com/bdunne) has found and [removed duplications in Aggregation mixin](https://github.com/ManageIQ/manageiq/pull/16368) and [Harpreet](https://github.com/h-kataria) has fixed button groups by [removing unneeded attribute](https://github.com/ManageIQ/manageiq-ui-classic/pull/2630) from toolbar definition.
+
+## Wrapping up
+
+So this was basically yet another awesome week in ManageIQ-land :wink:
+Don't forget to be awesome and happy! See you next time.
```
Comment (jntullo) [inline site/_posts/2017-11-06-call-me-gapri.md]: 😃

**Comment by cybette** [comment]
Looks great @romanblanco! If you can make the minor changes from above comments, I'll merge it after that. I'm swinging between ground level and 10km in the sky today so responses might be delayed.

File: site/_posts/2017-11-06-call-me-gapri.md (hunk 1 above)
Comment (romanblanco) [reply site/_posts/2017-11-06-call-me-gapri.md]: Sure, thanks 😄

**Comment by romanblanco** [comment]
Updated. Thanks for corrections @jntullo, @JPrause 👍

**Pull request merged**